nuxt/entities/draftLyrics.ts

```diff
@@ -0,0 +1,7 @@
+import { PersistedEntity, TimestampedEntity } from '@/entities/common';
+import { Lyrics } from '@/entities/lyrics';
+
+export interface DraftLyrics extends PersistedEntity, TimestampedEntity {
```

Comment:
I'd suggest moving this to `@/entities/lyrics` instead of having a separate thing here.

Reply:
Done